staff/asier/react-spotify-api-promises/design/scripts/spotify-api-1.0.0.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Spotify API client.
+ *
+ * @version 1.0.0
+ */
+
+let spotifyApi;
+
+(function () {
+    "use strict";
+    const baseUrl = "https://api.spotify.com/v1/"
+    const token = 'BQCt_qBCV1kEnGnXS9KgyZpo5AJvcwfo3o1gPE6RJGz09CaZSJ9MUMd1pAjNnV8JxGH2kPMhZqQaNLdWGyGSYAyTgsQLBkBQjhtGe2kFph1PF_Rai6cPo-4k5nRnT4pbj-OJVAw'
+    const headers = { Authorization: 'Bearer ' + token }
+
+    
+    spotifyApi = {
+        baseUrl,
+        
+        call: function (url) {
+    
+            return fetch(url , { headers })
+
+                .then(res => res.json())
+           
+        },
+        /**
+         * Searches artists by matching a text.
+         *
+         * @param {String} query - The text to search.
+         * @param {Function} handleResults - Handles the results.
+         * @param {Function} handleError - Handles an error.

```

Comment:
WARN params handleResults and handleError are no longer part of this implementation because promise is returned instead. so they should be removed from this doc / comments.

Reply:
SAME for the other spotifyApi methods.